Formula/gecode.rb

```diff
@@ -21,11 +21,13 @@ class Gecode < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "1f818c76e47ef63048f87fd2b75a6b379eef97b934079a427578c727e2fa21f2"
   end
 
+  depends_on "qt@5"
```

Comment:
This will break the Monterey bottle, since `qt@5` failed to build on Monterey.

Reply:
The current release of Gecode does not yet support the use of Qt6. Although we can fix this for an upcoming release, this would be a bit harder for now.

I am wondering about the problem that you are referring to. I am working on this on a M1 mac using Monterey and using `qt@5` seems to install correctly (and builds Gecode correctly). When does the problem occur?